src/flatMap.js

```diff
@@ -0,0 +1,3 @@
+export function flatMap(callback, thisArg) {
+	return this.map(callback, thisArg).flat();
```

Comment:
Might this implementation have unintended side effects when binding is used? The solution would be to return to `Array.prototype.map.apply(this, arguments).flat()`. Perhaps a test is needed to verify whether an overridden `this.map` is used or if `Array.prototype.map` is used?

Reply:
You're right, I forgot about bind arguments. I'll switch it back to use `arguments`.

Reply:
Done